lib/chef/resource/cron_d.rb

```diff
@@ -87,7 +87,7 @@ def self.validate_dow(spec)
       end
 
       property :cron_name, String,
-               description: "Set the name of the cron job. If this isn't specified we'll use the resource name.",
+               description: "An optional property to set the cron name if it differs from the resource block's name.",
```

Comment:
`cron entry name` maybe?

Reply:
These aren't actually entries. They're just standalone job definitions in cron.d